odl/trafos/util/ft_utils.py

```diff
@@ -625,13 +626,17 @@ def reciprocal_space(space, axes=None, halfcomplex=False, shift=True,
     recip_grid = reciprocal_grid(space.grid, shift=shift,
                                  halfcomplex=halfcomplex, axes=axes)
 
+    # Need to do this for axes of length 1 that are not transformed
+    non_axes = [i for i in range(space.ndim) if i not in axes]
+    min_pt = {i: space.min_pt[i] for i in non_axes}
```

Comment:
Why do we use a dict here?

Reply:
To leave the other axes alone.

In some sense I tend to think nowadays that this syntax of `min_pt={axis: value}` for `uniform_*` is overkill, and that one should specify full vectors or nothing. This was me half a year ago or so, trying to do fancy stuff, but it was already buggy back then, see #874  😇

Reply:
Please go ahead then!

Reply:
Not doing this now, since it might be useful after all. Without this possibility one would have to reverse engineer what's going on inside the calculation of min and max to provide the full vectors.